src/content/docs/en/editor-setup.mdx

```diff
@@ -32,7 +32,11 @@ import ReadMore from '~/components/ReadMore.astro';
 
 ## JetBrains IDEs
 
-Initial support for Astro landed in WebStorm 2023.1. You can install the official plugin through [JetBrains Marketplace](https://plugins.jetbrains.com/plugin/20959-astro) or by searching for "Astro" in the IDE's Plugins tab. This plugin includes features like syntax highlighting, code completion, and formatting, and plans to add even more advanced features in the future. It is also available to all other [JetBrains IDEs with JavaScript support](https://www.jetbrains.com/products/#lang=js&type=ide).
+Initial support for Astro landed in WebStorm 2023.1. You can install the official plugin through [JetBrains Marketplace](https://plugins.jetbrains.com/plugin/20959-astro) or by searching for "Astro" in the IDE's Plugins tab. This plugin includes features like syntax highlighting, code completion, and formatting. 
+
+Since version 2024.2, [Webstorm supports the Astro Language Server Protocol](https://blog.jetbrains.com/webstorm/2024/08/webstorm-2024-2/#astro-language-server-enabled-by-default), leading to improved code completion, better developer experience, and fixing several known issues. 
+
+You can toggle the language server in `Settings | Languages & Frameworks | TypeScript | Astro`.
```

Comment:
```suggestion

[Webstorm](https://www.jetbrains.com/webstorm/) is a JavaScript and TypeScript IDE that added support for the Astro Language Server Protocol (LSP) in version 2024.2. This plugin includes features like syntax highlighting, code completion, and formatting.

Install the official plugin through [JetBrains Marketplace](https://plugins.jetbrains.com/plugin/20959-astro) or by searching for "Astro" in the IDE's Plugins tab. You can toggle the language server in `Settings | Languages & Frameworks | TypeScript | Astro`.
```
I'm thinking this now feels more inline with documenting "how things are" vs. including "announcements" in official documentation that will very soon be dated. There was also some repetition between the two paragraphs (one saying features it had, the other saying these features were now better...)

Is there any nuance or accuracy I've lost here with this edit? How does this seem to you?

Reply:
This looks good to me!  I don't suppose there is much merit in including the link to the Webstorm change logs. I think I'm overly used to "citing my sources" but the way you've phrased it works well.

Rather than the news release, maybe we should include a link to their docs specifically regarding Astro? How would you feel about that?

For example:
[Webstorm Docs | Astro](https://www.jetbrains.com/help/webstorm/astro.html#ws_astro_configure_language_service)

Reply:
Just a note that "The Astro Language Server Protocol" is not accurate, the Astro language server is a language server implementing the language server protocol, but we don't own said protocol, right.

Reply:
Got it, I'll commit another revision tonight.

Reply:
Link to docs is perfectly legit in my books! Happy to have that included.

If linking inline text, just make sure the text link itself is clear what it is, where it goes etc. (e.g. [Webstorm Astro plugin docs] vs [here]). Thanks!